datafusion/physical-expr/src/functions.rs

```diff
@@ -69,14 +70,12 @@ pub fn create_physical_expr(
 
     let data_type = fun.return_type(&input_expr_types)?;
 
-    let fun_expr: ScalarFunctionImplementation =
-        create_physical_fun(fun, execution_props)?;
-
     let monotonicity = fun.monotonicity();
 
+    let fun_def = ScalarFunctionDefinition::BuiltIn(*fun);
```

Comment:
Here I just pass the `fun_def`, will call `create_physical_fun` when `execute` the `PhysicalExpr`